google-cloud-clients/google-cloud-datastore/src/main/java/com/google/cloud/datastore/FullEntity.java

```diff
@@ -62,7 +62,7 @@
     return new Builder<>(copyFrom);
   }
 
-  static FullEntity<?> fromPb(com.google.datastore.v1.Entity entityPb) {
+  public static FullEntity<?> fromPb(com.google.datastore.v1.Entity entityPb) {
```

Comment:
This isn't compiling. `Entity` extends `FullEntity` and it's `fromPb` method is still package private.

Reply:
Thank you for the review. Fixed it and rebased in https://github.com/googleapis/google-cloud-java/pull/5759/commits/e7146087c4715368caff098bc6f309c574fd2b7e.